bot/seasons/evergreen/trivia_quiz.py

```diff
@@ -0,0 +1,238 @@
+        """A function which sends the score."""
+        embed = discord.Embed(colour=discord.Colour.blue())
+        embed.title = "Score Board"
+        embed.description = ""
+        for k, v in player_data.items():
+            embed.description += f"{k} : {v}\n"
+        await channel.send(embed=embed)
+
+    @staticmethod
+    async def declare_winner(channel: discord.TextChannel, player_data: dict) -> None:
+        """A function declare the winner of the quiz."""
+        if player_data:
+            highest_points = max(list(player_data.values()))
+            no_of_winners = list(player_data.values()).count(highest_points)
+
+            # Check if more than 1 player has highest points.
+            if no_of_winners > 1:
+                winners = []
+                points_copy = list(player_data.values()).copy()
+                for _ in range(no_of_winners):
+                    index = points_copy.index(highest_points)
+                    winners.append(list(player_data.keys())[index])
+                    points_copy[index] = 0
+                winners_mention = None
+                for winner in winners:
+                    winners_mention += f"{winner.mention} "
+
+            else:
+                author_index = list(player_data.values()).index(highest_points)
+                winner = list(player_data.keys())[author_index]
+                winners_mention = winner.mention
+            await channel.send(
+                f"Congratz {winners_mention} :tada: "
+                f"You have won this quiz game with a grand total of {highest_points} points!!"
+            )
+
+    @property
+    def category_embed(self) -> discord.Embed:
+        """A function which returns an embed showing all avilable categories."""
+        embed = discord.Embed(colour=discord.Colour.blue())
+        embed.title = "The available question categories are:"
+        embed.description = ""
+        for cat, description in self.categories.items():
+            embed.description += f"**- {cat.capitalize()}**\n{description.capitalize()}\n"
+        embed.set_footer(text="If not category is chosen, then a random one will be selected.")
+        return embed
+
+    @staticmethod
+    async def send_answer(channel: discord.TextChannel, question_dict: dict) -> None:
+        """A function that sends the answer."""
+        answer = question_dict["answer"]
+        info = question_dict["info"]
+        embed = discord.Embed(color=discord.Colour.red())
+        embed.title = f"The correct answer is **{answer}**\n"
+        embed.description = ""
+        if info != "":
+            embed.description += f"**Information**\n{info}\n"
+        embed.description += "Lets move to the next question."
+        await channel.send(embed=embed)
+
+
+def setup(bot: commands.Bot) -> None:
+    """Loading the cog."""
+    bot.add_cog(TriviaQuiz(bot))
+    logger.debug("TriviaQuiz cog loaded!")
```

Comment:
```suggestion
    logger.debug("TriviaQuiz cog loaded")
```
Keep consistent with other cases of similar code in the project.